### ISSUE TITLE
What should we do about references to Regional Coordinators? 

### ISSUE THREAD

**Issue by maneesha** [carpentries/carpentries.org#232]

**Comment by maneesha** [comment]
From @OscarSiba 

> On workshops FAQ - "I do not see a Regional Coordinator for my area. Who should I speak with regarding a workshop?" - the link to this is broken. And again, not sure if this role still applies. For @sheraaronhurt or @danielle06 to confirm and/or rephrase.

**Comment by danielle06** [comment]
Thank you! We are in the process of updating the entire FAQ and we talked in WIT about leaving it as is until we have completed those updates and they are ready to be incorporated. Will leave this open as a reminder of what changes need to be made.

**Comment by maneesha** [comment]
@danielle06  and @sheraaronhurt  Do you have a timeline for when you want to come back to this?

**Comment by danielle06** [comment]
hey @maneesha , thank you for checking on this. I am working on the FAQs now (I believe that is where the references are) for the SOW fee project. I haven't put any PRs in yet, I'm just going through the changes that need to be made. Is it okay to start putting those in next week?

**Comment by maneesha** [comment]
Yes - whenever you are ready is fine.  I just don't want to lose track of things.

**Comment by maneesha** [comment]
@danielle06 Is there anything else related to this you are still working on? Otherwise I'd like to close this issue.

**Comment by maneesha** [comment]
@danielle06  Following up on my question above, just for reference - the question "What happens if I need to cancel or postpone a planned workshop?" on the [workshops FAQ](https://carpentries.org/workshops/workshops-faq/) page still references Regional Coordinators.

**Comment by danielle06** [comment]
@maneesha , great catch - I am planning to make other updates first thing tomorrow. I would like to keep this open so I remember to make the change to "What happens if I need to cancel or postpone a planned workshop?" I will close this after.
